src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -29,18 +29,22 @@ local tLines = {}
 local bRunning = true
 
 -- Colours
-local highlightColour, keywordColour, commentColour, textColour, bgColour, stringColour
+local highlightColour, booleanColour, keywordColour, functionColour, commentColour, textColour, bgColour, stringColour
 if term.isColour() then
-    bgColour = colours.black
-    textColour = colours.white
-    highlightColour = colours.yellow
-    keywordColour = colours.yellow
-    commentColour = colours.green
-    stringColour = colours.red
+    bgColour = settings.get("edit.backgroundColor") or colours.black
+    textColour = settings.get("edit.textColor") or colours.white
+    highlightColour = settings.get("edit.highlightColor") or colours.yellow
+    keywordColour = settings.get("edit.keywordColor") or colours.yellow
+    booleanColour = settings.get("edit.booleanColor") or colours.magenta
```

Comment:
Maybe "literal colour", so we can include `nil` too?

Reply:
Again, good idea.